Dockerfile

```diff
@@ -0,0 +1,213 @@
+    cd build ; \
+    cmake ../moab -DENABLE_HDF5=ON \
+                -DENABLE_NETCDF=ON \
+                -DBUILD_SHARED_LIBS=OFF \
+                -DENABLE_FORTRAN=OFF \
+                -DCMAKE_INSTALL_PREFIX=/MOAB ; \
+    make -j"$compile_cores" ; \
+    make -j"$compile_cores" install ; \
+    rm -rf * ; \
+    cmake ../moab -DBUILD_SHARED_LIBS=ON \
+                -DENABLE_HDF5=ON \
+                -DENABLE_PYMOAB=ON \
+                -DENABLE_BLASLAPACK=OFF \
+                -DENABLE_FORTRAN=OFF \
+                -DCMAKE_INSTALL_PREFIX=/MOAB ; \
+    make -j"$compile_cores" ; \
+    make -j"$compile_cores" install ; \
+    cd pymoab ; \
+    bash install.sh ; \
+    python setup.py install ; \
+    fi
+
+
+# Clone and install Double-Down
+RUN if [ "$include_neutronics" = "true" ] ; \
+    then git clone https://github.com/pshriwise/double-down ; \
+    cd double-down ; \
+    mkdir build ; \
+    cd build ; \
+    cmake .. -DCMAKE_INSTALL_PREFIX=.. \
+        -DMOAB_DIR=/MOAB \
+        -DEMBREE_DIR=/embree/lib/cmake/embree-3.12.1 \
+        -DEMBREE_ROOT=/embree/lib/cmake/embree-3.12.1 ; \
+    make -j"$compile_cores" ; \
+    make -j"$compile_cores" install ; \
+    fi
+
+# Clone and install DAGMC
+RUN if [ "$include_neutronics" = "true" ] ; \
+    then mkdir DAGMC ; \
+    cd DAGMC ; \
+    git clone -b develop https://github.com/svalinn/dagmc ; \
+    mkdir build ; \
+    cd build ; \
+    cmake ../dagmc -DBUILD_TALLY=ON \
+        -DCMAKE_INSTALL_PREFIX=/dagmc/ \
+        -DMOAB_DIR=/MOAB \
+        -DBUILD_STATIC_LIBS=OFF \
+        -DBUILD_STATIC_EXE=OFF ; \
+    make -j"$compile_cores" install ; \
+    rm -rf /DAGMC/dagmc /DAGMC/build ; \
+    fi
+
+# Clone and install OpenMC with DAGMC
+RUN if [ "$include_neutronics" = "true" ] ; \
+    then git clone --recurse-submodules https://github.com/openmc-dev/openmc.git /opt/openmc ; \
+    cd /opt/openmc ; \
+    mkdir build ; \
+    cd build ; \
+    cmake -Doptimize=on -Ddagmc=ON \
+        -DDAGMC_DIR=/DAGMC/ \
+        -DHDF5_PREFER_PARALLEL=on ..  ; \
+    make -j"$compile_cores" ; \
+    make -j"$compile_cores" install ; \
+    cd ..  ; \
+    pip install -e .[test] ; \
+    /opt/openmc/tools/ci/download-xs.sh ; \
+    fi
+
+ENV OPENMC_CROSS_SECTIONS=/root/nndc_hdf5/cross_sections.xml
+
+# Copies over the Paramak code from the local repository
+
+RUN if [ "$include_neutronics" = "true" ] ; \
+    then pip install neutronics_material_maker ; \
+    pip install parametric_plasma_source ; \
+    fi
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+
+# Copy over the source code, examples and tests
+COPY run_tests.sh run_tests.sh
+COPY paramak paramak/
+COPY examples examples/
+COPY setup.py setup.py
+COPY tests tests/
+COPY README.md README.md
+
+# using setup.py instead of pip due to https://github.com/pypa/pip/issues/5816
+RUN python setup.py install
+
+WORKDIR examples
```

Comment:
**MAINTAINABILITY**

Elaborating, the rationale behind this line `WORKDIR examples` is unclear. This causes issues when trying to run commands like `pytest tests` (also commented) and generally trying to use the project from inside the Docker container.

<p align="right">
    <a href="https://www.pullrequest.com/">
        <img width="15%" src="https://static.pullrequest.com/images/dashboard/pullrequest-logo.svg">
    </a>
</p>

Reply:
workdir in the dockerfile has now been removed